backend/src/server.py

```diff
@@ -207,6 +207,16 @@ async def run(request: Request):
         chain = parse_json(full_data["data"])
         optimize(chain)
 
+        # Remove all Generator values from the cache for each new run
+        # Otherwise, their state will cause them to resume from where they left off
```

Comment:
This makes no sense IMO. Why do you need to clear the cache? Are you telling me that iterators start iterating their first item when we run them in `/run/individual`? Because if that's the case, the problem isn't that their stale iterator is cached, but that they started iterating.

Reply:
I guess I never actually fully confirmed it, but it appeared to be staring from the last index whenever I stopped and restarted the chain. So let's say I had a load images node which got to 600/1000 and I stoppee the chain. When I pressed play again, it would go from 0-600 really quickly then continue going slower at 600.

So, I did this and confirmed it removed the nodes from the cache in case that was the issue. I never actually got to confirm it because my kill requests would end up restarting the executor backend which would reset it anyway.